components/monitoring/grafana/base/rbac/grafana-maintainers.yaml

```diff
@@ -0,0 +1,12 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: appstudio-grafana-maintainers
+  namespace: appstudio-grafana
+subjects:
+  - kind: Group
+    name: konflux-o11y-admins
```

Comment:
Here I proposed to restrict this to a subset of o11y team. This group currently has @amisstea @eisraeli and @ifireball . I can use the konflux-o11y group also if you want to grant these permissions to the entire team. Let me know what you want to do

Reply:
Its a very short list of people, and since we will all start supporting this domain I think we all should be granted with permission

Reply:
I do not mind changing this, I copied the pattern other groups use but that said, if o11y want to grant this to all members, works for me :)

Reply:
I see little issue with using the non-admin group in this instance since it's scoped to just grafana and not something with more sensitive data or in the critical path of the end user's workflow.

Reply:
Done, I used konflux-o11y team